scripts/tests/run_python_test.py

```diff
@@ -89,7 +90,34 @@ def DumpProgramOutputToQueue(thread_list: typing.List[threading.Thread], tag: st
 @click.option("--script-gdb", is_flag=True,
               help='Run script through gdb')
 @click.option("--quiet", is_flag=True, help="Do not print output from passing tests. Use this flag in CI to keep github log sizes manageable.")
-def main(app: str, factoryreset: bool, factoryreset_app_only: bool, app_args: str, script: str, script_args: str, script_gdb: bool, quiet: bool):
+@click.option("--load-from-env", default=None, help="YAML file that contains values for environment variables.")
+def main(app: str, factoryreset: bool, factoryreset_app_only: bool, app_args: str, script: str, script_args: str, script_gdb: bool, quiet: bool, load_from_env):
+    if load_from_env:
+        reader = MetadataReader(load_from_env)
+        runs = reader.parse_script(script)
+    else:
+        runs = [
+            Metadata(
+                py_script_path=script,
+                run="cmd-run",
+                app=app,
+                app_args=app_args,
+                script_args=script_args,
+                factoryreset=factoryreset,
+                factoryreset_app_only=factoryreset_app_only,
+                script_gdb=script_gdb,
+                quiet=quiet
+            )
+        ]
+
+    for run in runs:
+        print(f"Executing run: {run.py_script_path}")
```

Comment:
for multiple runs this will print out the same run several times.
Don't we have some discriminator somewhere? In your changed script this would be `run1`.

Ideally this should say something like `Executing TCS_SC_3_6.py run1` or just `Executing run1`

Reply:
Fixed.

Reply:
Fixed.